### DISCUSSION TITLE
How to do resizable images without stretching.

### DISCUSSION THREAD

**Discussion by MikeMorenoDSP** [alshedivat/al-folio#2526] [Q&A] [answered]
Hello,
First, thanks for the template, looks great.
How can I place images in a post that only shrink based on the browser's size.
I say this because, figure.liquid stretches my image, and I don't want that.
I only want my image to be shrunken if the browser size is smaller than the image.
Thanks for the help in advance.

---

EDIT: it worked, thanks. Here's an example:
```html
<div class="row" style="text-align: center;">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.liquid loading="eager" max-width="188px" max-heigth="307px" path="assets/img/posts/basic-waveforms/sawtooth.png" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
```

**Comment by george-gca** [comment, marked as answer]
There are 2 params for the `figure.liquid` that might help you: `max-width` and `max-height`. Give them a try.
